driver/pace/driver/grid.py

```diff
@@ -196,6 +199,82 @@ def get_grid(
         return damping_coefficients, driver_grid_data, grid_data
 
 
+@GridInitializerSelector.register("external")
+@dataclasses.dataclass
+class ExternalNetcdfGridConfig(GridInitializer):
+    """
+    Configuration for grid initialized from external data.
+    Input is from tile files generated by FRE-NCtools methods
+    The ExternalNetcdfGridConfig get_grid member method calls
+    the MetricTerms class method from_generated which generates
+    an object of MetricTerms to be used to generate the
+    damping_coefficients, driver_grid_data, and grid_data variables
+    We do not read in the dx, dy, or area values as there may be
+    inconsistencies in the constants used during calculation of the
+    input data and the model use.
```

Comment:
Suggest 'offline calculation of the grid inputs and those defined for use within the model'